robot_smach_states/src/robot_smach_states/topological_navigation.py

```diff
@@ -0,0 +1,223 @@
+# System
+import typing
+
+# ROS
+import rospy
+import smach
+
+# TU/e Robotics
+from robot_skills.robot import Robot
+from topological_action_planner_msgs.msg import Edge, Node
+
+# Robor Smach States
+from robot_smach_states.util.designators import Designator, EdEntityDesignator, EntityByIdDesignator
+from robot_smach_states.manipulation import OpenDoor
```

Comment:
This is a circular import: the manipulation module depends on the navigation module because the robot needs to drive to items to pick up. Now we turn this around